authN-authZ/auth-istio/README.md

```diff
@@ -45,7 +45,7 @@ The istio ingress gateway will be used to access the chatQnA service in differen
 
 Authentication and authorization are essential for securing microservices architectures. Using Bearer JWT tokens for these processes ensures that only authenticated users with valid tokens can access specific services, protecting sensitive data. Authentication verifies user identity, while authorization controls their permissions. This layered approach not only prevents unauthorized access but also provides detailed control over service interactions, maintaining system security and compliance. Here we leverage Istio mechanisms together with Bearer JWT tokens to fulfill that.
 
-<img src="./docs/OPEA auth flow with OIDC provider.png" width="700" height="300">
+![OPEA auth flow with OIDC provider](./docs/OPEA_auth_flow_with_OIDC_provider.png)
```

Comment:
Is the image size and layout changed if remove image width and height setting?
The width and height could be set with HTML.

Reply:
Yes, the image size changes but still looks OK without manually setting the width and height.